core/src/test/java/org/elasticsearch/common/io/StreamsTests.java

```diff
@@ -81,7 +81,7 @@ public void testCopyToString() throws IOException {
         String result = copyToString(in);
         assertThat(result, equalTo(content));
     }
-    
+
```

Comment:
Thanks!

Yeah, auto import causes a fair amount of pain like this.

There are a few options. You can use the diff feature in IntelliJ to revert specific lines back to their original version. Or, you can use `git add --interactive` to interactively add only the lines that you want to stage for commit leaving out the whitespace changes.